c/pthread-lit/fkp2014_true-unreach-call.i

```diff
@@ -654,25 +654,26 @@ void __VERIFIER_assert(int cond) {
   }
   return;
 }
-void __VERIFIER_atomic_assert(int cond) {
```

Comment:
You are removing `__VERIFIER_atomic_assert`, but it seems you did not introduce `__VERIFIER_assert` instead. Is that right?